### PR TITLE
Add Pulsar Community Weekly Update - 2020-10-24~ 2020-10-30

### PR DESCRIPTION
**Pull request by Huanli-Meng**

### CODE REVIEW

File: content/2020/2020-10/2020-10-30-pulsar-weekly.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+- [Broker] Add the `nettyMaxFrameSizeBytes` setting to the `standalone.conf` file.
+
+    https://github.com/apache/pulsar/pull/8385 ([@massakam](https://github.com/massakam))
+    
+- [Java Client] Disable the batch receiving timer for Readers.
+
+    https://github.com/apache/pulsar/pull/8381 ([@lhotari](https://github.com/lhotari))
+
+- [Broker] Allow the tenant Admin to loo kup topics.
```
Comment (Jennifer88huang) [inline content/2020/2020-10/2020-10-30-pulsar-weekly.md:35]: typo?

File: content/2020/2020-10/2020-10-30-pulsar-weekly.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+    https://github.com/apache/pulsar/pull/7250 ([@rdhabalia](https://github.com/rdhabalia))
+
+### Notable Bug Fix
+
+- [Broker] Fix the deadlock that occurred during topic ownership check.
+
+    https://github.com/apache/pulsar/pull/8406 ([@massakam](https://github.com/massakam))
+
+- [Java client] Fix the issue that newly-created consumers cannot be paused for subscribing topics.
```
Comment (Jennifer88huang) [inline content/2020/2020-10/2020-10-30-pulsar-weekly.md:81]: It seems that it's not the correct meaning of [this issue](https://github.com/apache/pulsar/issues/8214). If you need help, we can work on the description together.

File: content/2020/2020-10/2020-10-30-pulsar-weekly.md [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+
+    https://github.com/apache/pulsar/pull/8387 ([@aloyszhang](https://github.com/aloyszhang))
+
+- [Admin API] Validate the retention policy.
+
+    https://github.com/apache/pulsar/pull/8358 ([@lhotari](https://github.com/lhotari))
+
+- [Go client] Fix the memory leak issue for in the Golang client.
```
Comment (Jennifer88huang) [inline content/2020/2020-10/2020-10-30-pulsar-weekly.md:89]: remove "for"?

**Review by Jennifer88huang — COMMENTED** [review]
Left some comments, please check.
Overall, it's great job.

**Comment by Huanli-Meng** [comment]
@Jennifer88huang Updated docs based on your comments. PTAL.

**Review by Jennifer88huang — APPROVED** [review]

**Pull request merged**